lib/Sema/CSRanking.cpp

```diff
@@ -963,7 +963,7 @@ SolutionCompareResult ConstraintSystem::compareSolutions(
              ? SolutionCompareResult::Better
              : SolutionCompareResult::Worse;
   }
-  
+
```

Comment:
Didn't mean to leave this but I don't think I want to restart CI just to get rid of it